tools/ka-customizer-runner

```diff
@@ -1,11 +1,14 @@
 #!/bin/bash -ex
 
+BRANCH=$1
+shift
+
 if [ "$1" = "update" ]; then
     shift
-    flatpak update com.endlessm.Platform master
-    flatpak update com.endlessm.Sdk master
+    flatpak update "com.endlessm.Platform//$BRANCH"
+    flatpak update "com.endlessm.Sdk//$BRANCH"
```

Comment:
BRANCH is intentionally left blank in that case, to run the default branch for the app. I don't think there's an easy way to explicitly get the platform version a flatpak app has been built against, so its not easy to write the branch out explicitly at the moment. See discussion here https://phabricator.endlessm.com/T15173#321665